components/oneapi-validation/src/main/java/com/wso2telco/dep/oneapivalidation/service/impl/ussd/ValidateUssdSend.java

```diff
@@ -70,6 +71,11 @@ public void validate(String json) throws CustomException {
                 ussdAction = nullOrTrimmed(requestData.getString("ussdAction"));
             }
 
+            if (!(ussdAction.equals(MTINIT))) {
```

Comment:
Better to use the constant value in the front when checking equals. could be a possibility of throwing a null pointer exception if the "ussdAction" is null
https://stackoverflow.com/questions/11007781/java-null-string-equals-result

Reply:
done